src/Parser.h

```diff
@@ -297,7 +286,7 @@ inline std::string Parser<T>::romanize_segment(int segmentNbr) {
     if (tempItem != NULL) {
         temp += tempItem->romanization;
     } else {
-        temp += convert_trash_segment(segments[segmentNbr], true);
```

Comment:
shouln't it still be here for romanization (i.e  Chinese -> Pinyin) conversion ?